exporters/autoexport: add support for comma-separated values for OTEL_{METRICS,TRACES,LOGS}_EXPORTER (#5830)
open-telemetry/opentelemetry-go-contrib: CHANGELOG.md

- Add support to configure views when creating MeterProvider using the config package. (#5654)
- Add log support for the autoexport package. (#5733)
- Add support for disabling the old runtime metrics using the `OTEL_GO_X_DEPRECATED_RUNTIME_METRICS=false` environment variable. (#5747)

> - Add support for comma-separated values for `OTEL_TRACES_EXPORTER`, `OTEL_LOGS_EXPORTER`, `OTEL_METRICS_EXPORTER` to enable multiple exporters at once. (#5830)

Reviewer: ```suggestion
- Add support for comma-separated values for the `OTEL_TRACES_EXPORTER`, `OTEL_LOGS_EXPORTER`, `OTEL_METRICS_EXPORTER` environment variables in `go.opentelemetry.io/contrib/exporters/autoexport`. (#5830)
```
